README.md

```diff
@@ -42,7 +42,7 @@ Plug 'iamcco/markdown-preview.nvim', { 'do': { -> mkdp#util#install() }, 'for':
 
 
 " If you have nodejs and yarn
-Plug 'iamcco/markdown-preview.nvim', { 'do': 'cd app && yarn install'  }
+Plug 'iamcco/markdown-preview.nvim', { 'do': 'cd app && yarn install && npm install'  }
```

Comment:
`&& npm install` is no need.

Reply:
It should be `yarn install` or `npm install`

Reply:
Thanks. I tested it again and found that only ```npm install``` is need for msys2 platform. Now I have modified this pull request.